ansible_shed/shed.py

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env python3
 
 import asyncio
-import logging
-import re
-from git import Git, Repo
-from configparser import ConfigParser
 from collections import defaultdict
+from configparser import ConfigParser
+from git import Git, Repo
 from json import dumps
+import logging
 from pathlib import Path
 from random import randint
+import re
+import shutil
 from subprocess import CompletedProcess, PIPE, run
 from time import time
 from typing import Dict
```

Comment:
WTF is this sorting? 
- In python, most people go `import.*` alphabetically
  - Then `from.*` alphabetically
  - Lastly put local/thrid party modules with the same as above separation

Reply:
I was sorting by lib, happy to revert it.